src/MinHashSimilarities.java

```diff
@@ -13,14 +13,14 @@
     int numPerm;
     int intersect = 0;
     int union = 0;
-    int sameMH;
+    int sameMH=0;
```

Comment:
I think you want to declare this in the function that it is used in (`approximateJaccard`). Otherwise every time we call `approximateJaccard`, it will continue incrementing the variable using the last sum from the previous two files. Unless this was intended?